src/client/service/vaultsScan.ts

```diff
@@ -1,40 +1,37 @@
 import type { Authenticate } from '../types';
-import type { VaultManager } from '../../vaults';
+import type { NodeManager } from '../../nodes';
 import type * as nodesPB from '../../proto/js/polykey/v1/nodes/nodes_pb';
 import type * as grpc from '@grpc/grpc-js';
 import { utils as grpcUtils } from '../../grpc';
 import { utils as vaultsUtils } from '../../vaults';
+import { utils as nodesUtils } from '../../nodes';
 import * as vaultsPB from '../../proto/js/polykey/v1/vaults/vaults_pb';
 
 function vaultsScan({
-  vaultManager,
+  nodeManager,
   authenticate,
 }: {
-  vaultManager: VaultManager;
+  nodeManager: NodeManager;
   authenticate: Authenticate;
 }) {
   return async (
     call: grpc.ServerWritableStream<nodesPB.Node, vaultsPB.List>,
   ): Promise<void> => {
     const genWritable = grpcUtils.generatorWritable(call);
-    // Const nodeId = makeNodeId(call.request.getNodeId());
-
     try {
       const metadata = await authenticate(call.metadata);
       call.sendMetadata(metadata);
-
-      const vaults = await vaultManager.listVaults();
-      vaults.forEach(async (vaultId, vaultName) => {
+      const nodeId = nodesUtils.makeNodeId(call.request.getNodeId());
+      const list = await nodeManager.scanNodeVaults(nodeId);
```

Comment:
@tegefaulkes this should still be part of the `vaultManager` and not `nodeManager`.

Reply:
This is being addressed in #310.

Reply:
So if #310 moves this feature into `vaultManager`, I think that `VaultManager` will be using the `NodeConnection` features to actually call the scan vaults. Basically there shouldn't be a method called `scanNodeVaults` at all in `NodeManager`. If you're removing that code though, make sure to post it as as comment here, so I can refer to it.